docs/02-Install.md

```diff
@@ -105,7 +65,51 @@ In the LORIS front-end, under the Admin menu, go to the `Config` module. Under
 
 Click 'Submit' at the end of the Configuration page to save any changes.
 
-#### 2.4.5 FAQ
+#### 2.3.5 FAQ
 
 For common errors and frequently asked questions, please refer to the [Appendix
-  A](AppendixA-FAQ.md).
\ No newline at end of file
+  A](AppendixA-FAQ.md).
+
+
+## 2.4 Pipeline flow
+
+The pipeline was initially designed for **raw DICOM MRI data**, collected by a
+  longitudinally-organized multi-site study with a defined imaging acquisition
+  protocol. With modifications and further customization, it can handle any
+  modality of imaging data.
+
+Typically, images insertion into LORIS is performed via the following steps:
+
+1. Upload of a compressed set of the DICOM study via the **Imaging Uploader**
+     module (or transfer the compressed DICOM studies directly on the server
+     hosting LORIS).
+2. DICOM insertion into the `tarchive` tables in order to be able to see the
+     information stored in the DICOMs via the **DICOM Archive** module.
+3. Conversion of the DICOMs into MINC files for protocol validation and
+     insertion into the LORIS database. By default it will also create NIfTI
+     images from the MINC files and insert them into LORIS. (See the
+     [Configuration](#2.2-configuration) section above if you want to disable
+     this option.) One of two possible actions will follow depending on the
+     study-defined protocol:
+     
+    a. If a scan matches one of the protocol defined in the `mri_protocol`
+         table, then the MINC image will be stored into the `files` tables.
+         This inserted image is then accessible via the **Imaging Browser**
+         module and can be displayed in 3D using **BrainBrowser**.
+
+    b. If a scan does not match any of the protocol defined in the
+         `mri_protocol` table, then the MINC image of this scan will be stored
+         into the `mri_protocol_violated_scans` table. This violated image is
+         then accessible via the **MRI Violated Scans** module and can be
+         displayed in 3D using **BrainBrowser**.
+
+
+The graph below shows the different modules mentioned above with the
+  representation of the group of tables described in more details in the
+  [Technical Infrastructure](03-TechnicalInfrastructure.md) section. In
+  addition, the graph shows the name of the main script that is used to insert
+  the images into the LORIS database. More details about those scripts can be
+  found in the section [4 Scripts](04-Scripts.md).
```

Comment:
I would say in the Scripts section to be consistent with 2 lines earlier when you refer to section 3.

Reply:
addressed in https://github.com/aces/Loris-MRI/pull/273